### PR TITLE
add basic overseer stats to the facet admin dashboard

### PR DESCRIPTION
**Pull request by bnchdrff**
closes steemit/overseer#18

not real funnel tracking queries yet

- store all potential checkpoints in a constants file
- add an overseer query & a query builder to services
- modify admin dashboard to fetch the new data
- display data in a new card with a rangepicker widget
- note in the UI that the query is simply totals over a timespan, not a true funnel query

todo in the near future (maybe this sprint):
- ask overseer to execute the query rather than going straight to influxdb

todo in the distant future:
- write a real funnel query (see https://github.com/influxdata/influxdb/issues/191 and https://github.com/influxdata/influxdb/issues/9821)

requires a new env var `INFLUXDB_URL`

### CODE REVIEW

File: helpers/services.js [hunk 1]
```diff
@@ -233,6 +234,92 @@ function recaptchaRequiredForIp(ip) {
...
+    if (!response.results) {
+        throw new Error('influxdb query error');
+    }
+
+    let mappedResults;
+    try {
+        mappedResults = response.results.reduce(
+            (acc, cur) => ({
+                ...acc,
+                [cur.series[0].columns[1]]: cur.series[0].values[0][1] || null,
```
Comment (gl2748) [inline helpers/services.js]: it seems that `cur` object does not always have the `series` key...

e.g. adding console logs like this:
```
    try {
        mappedResults = response.results.reduce(
            (acc, cur) => {
                console.log('cur:', cur)
                return ({
                    ...acc,
                    [cur.series[0].columns[1]]: cur.series[0].values[0][1] || null,
                })
            },
            {}
        );
    } catch (error) {
        console.log('error:', error)
        throw new Error('influxdb data error');
    }
```

```
cur: { statement_id: 3 }
error: TypeError: Cannot read property '0' of undefined
    at response.results.reduce (/home/iain/faucet/helpers/services.js:295:28)
    at Array.reduce (<anonymous>)
    at Object.getOverseerStats (/home/iain/faucet/helpers/services.js:290:42)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

**Comment by gl2748** [comment]
LGTM.

File: db/config/config.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "use_env_variable": "DATABASE_URL",
     "operatorsAliases": false,
     "dialectOptions": {
-      "ssl": true
+      "ssl": false
```
Comment (jnordberg) [inline db/config/config.json]: Intentional?

Comment (bnchdrff) [reply db/config/config.json]: oops, no not intentional

**Pull request merged**